aem-authoring-toolkit-samples/toolkit-samples-package/src/main/content/jcr_root/apps/authoring-toolkit/samples/components/content/abilities-component/content/elements-list.html

```diff
@@ -0,0 +1,7 @@
+<template data-sly-template.elements="${@ data}">
+    <button id="warrior-abilities-button">ELEMENTS</button>
```

Comment:
is it safe to use `id`s in the components? what if the component is added to the page multiple times?